pkgs/servers/web-apps/livebook/default.nix

```diff
@@ -0,0 +1,43 @@
+
+{ lib, beamPackages, makeWrapper, rebar3, elixir, erlang, fetchFromGitHub }:
```

Comment:
Nit: Let's remove this empty line here

Reply:
Oops yeah, don't know how that snuck in. Fixed now :-)